build_from.py

```diff
@@ -95,5 +143,13 @@ def generate_diffs(reg_tree, act_title_and_section, builder, layer_cache):
     parser.add_argument('--checkpoint', required=False,
                         help='Directory to save checkpoint data')
 
+    parser.add_argument('--operation', action='store')
```

Comment:
Agreed on the complexity being unneeded at this time. We've slowly added a few other helper scripts, too; maybe a generic "eregs.py" runner would make more sense.

```bash
$ ./eregs full-pipeline my.xml --title 12
$ ./eregs notice-order --title 12 --part 1026
```
etc. I'd strongly recommend against tacking that on to `build_from`